README.md

```diff
@@ -26,7 +26,7 @@ Browser extension that adds configurable keyboard shortcuts to Google search, Yo
 ## Features
 
 - Lightweight
-- Supports Chrome, Firefox, and pre-Chromium Edge
```

Comment:
I intentionally phrased it this way because the version you uploaded supports pre-Chromium Edge, right?
I assumed users would know that it supports the new Edge, since it's based on Chrome (the same is true for Vivaldi, Brave, and any other Chromium-based browser).

Reply:
Previously, this was correct. The extension was however automigrated to the Chromium-based Edge's Add-ons. The previous URL even forwarded to the new store as well  The pre-Chromium Edge is rapidly being deprecated.

The new Edge however retains an own Add-ons store, even though extensions can also be installed from the Chrome Web Store. That does require an additional user permission. Therefore I propose also maintaining the extension in the Microsoft Edge Add-ons store.